templates/backstage/sidecar-setup.yaml

```diff
@@ -1,34 +1,27 @@
 ---
 containers:
-  - env:
-      - name: RCS_CONFIG_FILE
-        value: /app-root/config/rcsconfig.yaml
-      - name: RHDH_CONFIG_FILE
-        value: /app-root/config/sed.edit.RHDH_CONFIG_FILENAME
-      - name: PROJECT
-        value: rhdh
-    envFrom:
-      - secretRef:
-          name: sed.edit.RHDH_SECRETS_NAME
-    image: sed.edit.RCS_IMAGE
-    name: road-core-sidecar
+  - envFrom:
+    - secretRef:
+        name: llama-stack-secrets
+    image: 'quay.io/redhat-ai-dev/llama-stack:latest'
```

Comment:
It would be nice if this is configurable, I want to run my image with specific run.yaml